src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
+spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False&allowPublicKeyRetrieval=true

```

Comment:
I don't know if it's a good idea to add "&allowPublicKeyRetrieval=true' - it's a security breach after all.
While working on local  code, you have to use it, but I wouldn't put it on the main brench.